README.md

```diff
@@ -37,3 +37,5 @@ public class ClothesApiController {
 ## Github Actions
 
 - [Environment Variables](https://docs.github.com/en/actions/reference/environment-variables)
+
+shoes
```

Comment:
@NagaVijaya / @balajiachanta we think this is a typo, could you check

Reply:
updated read me per comment